fsd_utils/mapping/application/application_utils.py

```diff
@@ -44,8 +45,8 @@ def format_answer(answer):
             return [a.replace("'", "") for a in answer if isinstance(a, str)]
 
         return answer
-    except Exception as e:
-        current_app.logger.error(f"Could not format the answer, {e}")
+    except Exception:
+        current_app.logger.error(f"No formatting required for an answer: {answer}")
```

Comment:
Do we not want to know the exact exception in this case? This seems generic. I'd probably go with a combo of the two:

```
    except Exception as e:
        current_app.logger.error(f"No formatting required for answer: {answer}, exception {e} encountered")
```

Reply:
> Do we not want to know the exact exception in this case? This seems generic. I'd probably go with a combo of the two:
> 
> ```
>     except Exception as e:
>         current_app.logger.error(f"No formatting required for answer: {answer}, exception {e} encountered")
> ```

If the exception is always expected to be the same, then this can be disregarded

Reply:
This was already in main and I know @ramsharma-prog is reworking this area anyway so I'll leave it for now.